.github/workflows/test.yaml

```
@@ -61,7 +61,7 @@ jobs:
           - python-version: "3.11"
             go-version: "1.20"
           - python-version: "3.12"
-            go-version: "1.21"
+            go-version: "1.20"
```

```suggestion
            go-version: "1.21"
```

Restores an unrelated change